strawberryfields/backends/fockbackend/ops.py

```diff
@@ -415,6 +416,22 @@ def squeezing(r, theta, trunc, save=False, directory=None):
     return ret
 
 
+@functools.lru_cache()
+def two_mode_sqeezing(r, theta, trunc):
+    r"""The two-mode squeezing operator :math:`S_2(re^{i\theta})`.
+
+    Args:
+            r (float): two-mode squeezing magnitude
+            theta (float): two-mode squeezing phase
+            trunc (int): Fock ladder cutoff
+    """
+    ret, _, _ = S2gate(r, theta, cutoff=trunc)
+
+    ret = np.transpose(ret, [0, 2, 1, 3])
```

Comment:
I suppose this requires a new PR to the walrus, so that the transposition is no longer needed? @nquesada

Reply:
Yes, it would require a new PR to the Walrus. And I definitely agree, it would be great to do some timing comparisons to see the gain.